client/rpc/src/state/mod.rs

```diff
@@ -360,10 +379,122 @@ impl<Block, Client> State<Block, Client>
 			}.boxed()
 		})?;
 
+		module.register_subscription(
+			"state_subscribeRuntimeVersion",
+			"state_unsubscribeRuntimeVersion",
+			|_params, mut sink, ctx| {
+				let executor = ctx.executor.clone();
+				let client = ctx.client.clone();
+
+				let mut previous_version = client.runtime_version_at(&BlockId::hash(client.info().best_hash))
+					.expect("best hash is valid; qed");
+				let _ = sink.send(&previous_version);
+				let rt_version_stream = client.storage_changes_notification_stream(Some(&[StorageKey(well_known_keys::CODE.to_vec())]), None, )
+					.map_err(|blockchain_err| Error::Client(Box::new(blockchain_err)))
+					.map_err(to_jsonrpsee_call_error)?;
+
+				let fut = async move {
+                    rt_version_stream
```

Comment:
identation looks off here

Reply:
No, I think this one is fine (I do miss rustfmt though).